.changeset/moody-lies-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+"web-fragments": minor
```

Comment:
```suggestion
"web-fragments": patch
```

we've just gone for all `patch`es for now I believe, for this early stage maybe we'd want to keep it like that? 🤔 

(also I am not sure if this is a minor 🤔)